packages/ckeditor5-autosave/tests/manual/autosave.js

```diff
@@ -25,14 +33,6 @@ ClassicEditor
 		destroyButton.addEventListener( 'click', () => editor.destroy() );
 
 		const autosave = editor.plugins.get( Autosave );
-		autosave.adapter = {
```

Comment:
Used config callback instead of adapter because there is no word about adapter anywhere in our docs or guides, it is just mentioned somewhere deeper in API docs. I believe this is forgotten/not recommended way to integrate autosave.

Funny, with this integration, the problem with calling autosave during initialization does not happen (because the adapter is added after the editor is initialized).